### ISSUE TITLE
[FFMPEG] - Cannot init CUDA

### ISSUE THREAD

**Issue by RCFilm** [bassrock/plex-nvidia-docker#2]
Not sure what the issue is.
I am able to run nvidia-smi from unraid ssh and from within your plex-nvidia-docker, but when I try to run a process like FFmpeg with nvec I get Cannot init CUDA.

Little history... I was able to use the plugin and it worked great then after a reboot nothing.
I have tried changing back to stock then reinstall, no luck. 

kinda stumped.

+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
# nvidia-smi
Thu Dec 27 23:42:58 2018
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 410.78       Driver Version: 410.78       CUDA Version: 10.0     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  Quadro P400         On   | 00000000:06:00.0 Off |                  N/A |
| 34%   30C    P8    N/A /  N/A |      0MiB /  2000MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+

**Comment by RCFilm** [comment]
OK nevermind got it.
for some reason the "/boot/config/nvidia.sh" was rm. i added it manually and then added to the boot config per your step 12 & 13..

Definitely not perfect instructions yet lol :) 

Great work man

**Comment by RCFilm** [comment]
No, I have not had any segfaults. I also have been using the GRU as a --device, not rw path. Seems to work great for encoding for plex docker and FFmpeg inside plex docker using ( https://github.com/RCFilm/plexEncode ). Decoding does not work on plex, I have not tried to decode with FFmpeg inside yet.

I would like to get this up and running for h265ize, but having a little trouble with this build ( https://github.com/RCFilm/h265ize )
 
Here is my system and GPU card used
M/B: TYAN - FT48-B8812
CPU: 4x AMD Opteron™ 6274 @ 2200 - 64 Core
HVM: Enabled
IOMMU: Enabled
Cache: 768 kB, 16384 kB, 12288 kB
Memory: 60 GB (max. installable capacity 60 GB)
Network: bond0: load balancing (xor), mtu 1500 
Graphics Card 1: | Quadro P400
Kernel Drivers 1: | nvidia v410.78
Kernel Modules 1: | nvidia_drm, nvidia

**Comment by bassrock** [comment]
Interesting please keep me up to date on uptime and if any segfaults occur. I could get about 12-24hr uptime before a segfault.

**Comment by RCFilm** [comment]
Had a mishap when I installed DVB drivers and lost my Nvidia drivers so I removed than was going to reinstall your Nvidia plg but could not find it anymore.
lost the backup of it as-well when i installed the chmb version. the problem with that version is it only gives 6.6.1 unlike yours of 6.6.6
https://raw.githubusercontent.com/bassrock/Unraid-NVIDIA-Plugin/master/plugins/UnraidNVIDIA.plg
any help with a new link?

**Comment by bassrock** [comment]
Actually, we merged the repo and are in the progress of trying to get something to fully work. Because of that my version isn't live as we work to push forward. Sorry about that :/

**Comment by bassrock** [comment]
Hopefully when it is done unraid will be using https://github.com/NVIDIA/nvidia-docker
